app/src/main/res/layout/item_top_track.xml

```diff
@@ -0,0 +1,37 @@
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
WARNING: [UnusedResources] The resource `R.layout.item_top_track` appears to be unused
 Unused resources make applications larger and slow down builds.